packages.txt

```diff
@@ -0,0 +1,203 @@
+##############################################################################
+# ROCK STAR APPS:
+##############################################################################
+# Education - math games
+tuxmath
+# Education - typing tutor and games
+tuxtype
+tuxtype-data
+# Education - simple drawing program for young children
+tuxpaint
+tuxpaint-data
+tuxpaint-plugins
+tuxpaint-stamps
+
+## Minecraft (careful about versions...minecraft-pi basic but adequate?
+# Minetest requires GL driver that messes up VLC etc?),
+# Games - Minecraft (basic edition, built for pi)
+minecraft-pi
+# Games - MineTest is similar to Minecraft, but more advanced than minecraft-pi. Can play networked games on IIAB server.
+### Note: If graphics performance is slow, enable "Open GL Driver" as follows:
+###   - Open Terminal
+###   - Run "sudo raspi-config"
+###     - Advanced Options
+###     - GL Driver
+###     - GL (Full KMS)
+###
+### Note: Enabling Open GL Driver can make VLC video playback choppy
+minetest
+minetest-server
```

Comment:
```suggestion
```

Reply:
Commented this out with commit [0c45905](https://github.com/kidsoncomputers/kocifier/pull/20/commits/0c45905f47ee2397d642a504d345f38869dea45b)